packages/captp/src/captp.js

```diff
@@ -110,7 +119,7 @@ export const makeCapTP = (
 
   /** @type {any} */
   let unplug = false;
-  const quietReject = async (reason = undefined, returnIt = true) => {
+  const quietReject = (reason = undefined, returnIt = true) => {
```

Comment:
Why remove the `async`? AFAICT, it should only make a difference if the body synchronously throws. In that case, would you prefer to propagate a throw or a rejection?

It is a common convention that a function whose only successful results are promises indicates problems as rejections rather than throws, which `async` reliably enforces.

Reply:
If I had to guess it's to make sure we don't create an unhandled rejection. The promise that we return is handled, but it can't be if this function is async, because the rejection would propagate to the different promise created intrinsically by the async function.

Reply:
Yes, @mhofman is correct.  That's what the "quiet" in "quietReject" refers to.

Reply:
resolving